apps/omg_eth/lib/omg_eth/root_chain.ex

```diff
@@ -85,6 +85,7 @@ defmodule OMG.Eth.RootChain do
     {:array, {:tuple, [:bool, {:uint, 256}, {:bytes, 32}, :address, {:uint, 256}, {:uint, 256}]}}
 
     # solidity does not return arrays of structs
+    # following is tuple representing the struct in solidity: https://git.io/JfEi5
```

Comment:
Wow `git.io` - nice to know the tricks :)
...but as it's a comment and may stay here for some long time - are we aware of data retention of this shortener service?

Reply:
I've updated anyway, the url is a bit long though....

I can't find the data retention policy of this service but as it is native GitHub Service I would not be too much concern though...